notifications/defaultPreferences.ts

```diff
@@ -1,18 +1,26 @@
-import { NotificationPreferences } from '../graphql/types/preferences';
+type EmailChannel = { email: boolean };
+type Channel = { [channel: string]: boolean };
+export type Preference = { [category: string]: EmailChannel };
 
-export const DEFAULT_PREFERENCES: NotificationPreferences = {
-    chat: { email: true },
+export const DEFAULT_PREFERENCES: Preference = {
+    chat: { email: false },
+    survey: { email: false },
+    appointment: { email: false },
+    advice: { email: false },
+    suggestion: { email: false },
+    announcement: { email: false },
+    call: { email: false },
+    news: { email: false },
+    event: { email: false },
+    request: { email: false },
+    alternative: { email: false },
+};
+
+export const FIX_PREFERENCES: Preference = {
```

Comment:
Maybe we should not call it ENABLED_NOTIFICATIONS or FORCED_TYPES

Reply:
would it be better to have an export with all preferences already merged so we don't need to merge them in common/notification/index.ts?